release/release_tests.yaml

```diff
@@ -2552,6 +2552,37 @@
       cluster:
         cluster_compute: 2gpus_64cpus_gce.yaml
 
+- name: rllib_learning_tests_halfcheetah_appo_torch
+  group: RLlib tests
```

Comment:
Do we have to use this in the future for elaborate algorithm tests?

Reply:
Not sure what you mean?

This will run nightly now under the release pipeline, so it'll be covered and we won't miss if we break stuff.